addOns/kotlin/src/test/java/org/zaproxy/zap/extension/kotlin/VerifyScriptTemplates.java

```diff
@@ -19,24 +19,31 @@
  */
 package org.zaproxy.zap.extension.kotlin;
 
-import java.io.Reader;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.*;
```

Comment:
I'm sure this is likely an IDE setting, but can you use specific imports throughout?

Reply:
pretty sure i managed to get all these 😅